src/components/navbar/Navbar.jsx

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import { FaUserCircle } from "react-icons/fa";
+import { GiHamburgerMenu } from "react-icons/gi";
+import {
```

Comment:
export and import icons from an assets/icon.js folder and use it across your app